spec/shared_examples/user_reset_password_shared_examples.rb

```diff
@@ -314,17 +316,27 @@
       end
     end
 
-    it 'when change_password! is called, deletes reset_password_token' do
+    it 'when change_password is called, deletes reset_password_token' do
```

Comment:
This technically changes test functionality, I think leaving it as `change_password!` here is reasonable.